src/ui/actions_main_area.ui

```diff
@@ -146,30 +176,43 @@
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
+           <property name="toolTip">
+            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This control specifies where the tool bar should be located/docked with options to be positioned to the left, right or top of the main Mudlet screen.  There is a further option that allows the toolbar to be floated free and positioned any where on the desktop, and for a multi-screen (virtual) desktop even on a different monitor.&lt;/p&gt;&lt;p&gt;A &amp;quot;docked&amp;quot; toolbar can be undocked and dragged to a new position, it will automatically redock if that position is close to a valid docking area on screen or left floating otherwise; this will be remembered and reflected in the choice displayed here afterwards.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
Ah, blast, that tooltip reflects code that I was experimenting with but abandoned for this rush job (those ui files and the IDE processing of them do not lend themselves to git gui/diff work as the IDE will happily reorder things at the slightest whim) as it happens the current code as presented here __does forget__ manual toolbar relocation by the user of floating toolbars, it reverts to the saved left/top/right or floating state every time you save/touch anything on the Buttons editor - which is bloody user-hostile IMHO.  Also you can dock a floating toolbar in the __bottom__ dock area as we never prevented that area from being used, we only prevented it being specified.